### PR TITLE
SE-186 Use Parquet instead of CSV in TPC-H

### PR DESCRIPTION
**Pull request by alifarhid**
JIRA ticket:
https://bodo.atlassian.net/browse/SE-186

### CODE REVIEW

File: notebooks/tpch.ipynb [hunk 1]
```diff
@@ -9,54 +9,35 @@
     "TPC-H is a decision support benchmark that offers business-oriented ad hoc queries.\n",
```
Comment (hadia206) [inline notebooks/tpch.ipynb:9]: <p>do you know why some ranks are not printing?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Bodo-inc/Bodo-examples/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (hadia206) [inline notebooks/tpch.ipynb:9]: <pre class="ql-syntax" spellcheck="false">Line #13.        jn5 = jn4.merge(orders, left_on="L_ORDERKEY", right_on="O_ORDERKEY")
</pre><p>Thanks for remembering to update :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Bodo-inc/Bodo-examples/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

**Review by hadia206 — APPROVED** [review]
One minor comment. Otherwise looks good to me.

File: notebooks/tpch.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (alifarhid) [reply notebooks/tpch.ipynb:9]: <p>Updated it:</p><p>Dataset size is 700MB in parquet format. The original data was in CSV format and 2GB.</p><p> </p><p>There's a larger dataset available on "s3://bodo-example-data/tpch/s4/" which is 4GB in CSV. To see how you can read CSV version, refer to this [example](https://github.com/Bodo-inc/Bodo-examples/blob/master/hosted-trial-examples/tpch.ipynb)</p><p> </p><p> </p>

Comment (alifarhid) [reply notebooks/tpch.ipynb:9]: <p>all ranks are printing. It's Review NB's rendering problem. </p>

**Pull request merged**